src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -82,8 +82,18 @@ def __init__(
             unit_map=None):
         self.unit = unit
         self.result_type = result_type
-        self.unit_map = unit_map or dict(B=1, kB=1024, MB=1024 * 1024, GB=1024 * 1024 * 1024,
-                                         TB=1024 * 1024 * 1024 * 1024)
+        self.unit_map = unit_map or dict(
+            B=1,
+            kB=1024,
+            MB=1024 *
+            1024,
+            GB=1024 *
+            1024 *
+            1024,
+            TB=1024 *
```

Comment:
revert this change

Reply:
Looks like your env setting made a lot of unnecessary change, I would suggest you to revert those changes to avoid any surprises.